src/utilities/max.ts

```diff
@@ -0,0 +1,9 @@
+const { toArray } = require('./to-array');
+/**
+ *
+ * @param values
+ * @returns {number}
+ */
+export function max(values: Record<string, number> | number[]): number {
```

Comment:
I think `values` should be `number|number[]` here, since we are not dealing with strings here.

Reply:
@mubaidr the `string` refers to the key, the `number` is the value - this is due to the following unit test. The `string` refers to the object's keys (`a`, `b`, `c`, `d`) 

`__tests__/utilities/to-array.js`

```js
const { toArray } = require('../../src/utilities/to-array');

describe('to-array', () => {
  test('should convert object to array', () => {
    const obj = { a: 1, b: 5, c: 10, d: 0 };

    expect(toArray(obj)).toBeInstanceOf(Float32Array);
    expect(toArray(obj).length).toBe(4);
  });
});
```

Reply:
Oh ok. I missed that.